landlab/components/stream_power/cfuncs.pyx

```diff
@@ -4,6 +4,9 @@ cimport cython
 
 #from libc.math cimport fabs
 
+# as best as KRB can tell, the function _brentq is in c.
+# not sure what, if anything I need to do to 
+from scipy.optimize._zeros import _brentq as brentq
 
```

Comment:
Python docs says "Calling NumPy/SciPy functions currently has a Python call overhead; it would be possible to take a short-cut from Cython directly to C. (This does however require some isolated and incremental changes to those libraries; mail the Cython mailing list for details)."

Not sure what this is getting at, process wise. The easiest way (though kind of a crappy option) might be to find the function in scipy, get hold of its Cython code, and just paste the function part into our cfuncs...?